cusub_perception/detection_tree/src/detection_tree_node.cpp

```diff
@@ -184,14 +190,24 @@ void DetectionTree::darknetCallback(const darknet_ros_msgs::BoundingBoxesConstPt
     }
 }
 
+bool DetectionTree::checkIllegalDetection(int image_height, int image_width, darknet_ros_msgs::BoundingBox& box)
```

Comment:
I think FakeYolo is not supposed to be capably of generating these but I don't remember.

Reply:
had to add that because when we'd get close to jiangshi, then the image would cut out making the bounding box smaller and our estimated distance would increase so we'd think we were far away

Reply:
Good check to do, with optical flow I do the same check since the transform can send the box out of the image